ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -124,13 +128,23 @@ func (p *LocalProject) ReadTaskDefinition() error {
 			return err
 		}
 	} else if filename != "" {
+		filename, err = filepath.Abs(filename)
```

Comment:
nit: Should we name this something else like `absPath` since it's not just the filename anymore

Reply:
I did this to avoid the annoying there where err is defined earlier but if i give the path a new variable it gets clunky to assign stuff. It's a bit of a hack, but I thought it'd be okay since we won't need the original filename anymore.